frontend/components/landing/LandingSplash.vue

```diff
@@ -6,14 +6,14 @@
       <h1
         class="w-10/12 text-3xl font-bold leading-snug text-center lg:font-bold sm:text-4xl md:text-5xl xl:text-6xl md:max-w-lg xl:max-w-2xl sm:leading-snug md:leading-snug xl:leading-snug font-display"
       >
-        {{ $t("landing-header") }}
+        {{ $t("components.landing.splash.header") }}
```

Comment:
Note that given naming conventions this should be `components.landing.landing-splash.header` :) Similarly below.

Reply:
yes, I'll fix that. It's just that I had thought that the "landing" would be very repetitive, but it's better to leave it complete anyway